### PR TITLE
Feat/303 previous name

### PR DESCRIPTION
**Pull request by harshil1793**
## Description

Added previous names page with conditional full name field.

## Original issue(s)
department-of-veterans-affairs/va-forms-system-core#303

## Screenshots
img 1
img 2

## Definition of done
- [ ] Documentation has been updated, if applicable
- [ ] A link to the original github issue has been provided
- [ ] No sensitive information (i.e. PII/credentials/internal URLs etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: examples/burial-form/PreviousNames.jsx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import {
+  FullNameField,
+  Page,
+  RadioGroup
+} from '@department-of-veterans-affairs/va-forms-system-core';
+import { useFormikContext } from 'formik';
+import { ExpandingGroupClass } from '../Constant';
+
+export default function PreviousNames(props) {
+  useEffect(() => {
```
Comment (bjmccotter7192) [inline examples/burial-form/PreviousNames.jsx:11]: Can we remove this? I think it was causing stuttering in Firefox.

**Review by bjmccotter7192 — COMMENTED** [review]
Good idea with the expanding class, left one comment but otherwise looks good.

File: examples/burial-form/PreviousNames.jsx (hunk 1 above)
Comment (harshil1793) [reply examples/burial-form/PreviousNames.jsx:11]: you mean useEffect?

**Review by bjmccotter7192 — APPROVED** [review]
👍

**Pull request merged**